specification/app/resource-manager/readme.md

```diff
@@ -65,11 +65,33 @@ input-file:
   - Microsoft.App/preview/2024-02-02-preview/Subscriptions.json
   - Microsoft.App/preview/2024-02-02-preview/Usages.json
   - Microsoft.App/preview/2024-02-02-preview/FunctionsExtension.json
+  - Microsoft.App/preview/2024-02-02-preview/SessionPools.json
+  - Microsoft.App/preview/2024-02-02-preview/Sessions.json
 directive:
   - suppress: PatchBodyParametersSchema
     from: JavaComponents.json
     reason: |
       Java Component is using componentType as the discriminator. While the discriminator is a required property, this rule prevent it being present in the patch request body.
+  - suppress: LroErrorContent
+    from: SessionPools.json
+    reason: |
+      Using the same error response as other APIs.
+  - suppress: EnumInsteadOfBoolean
+    from: SessionPools.json
+    reason: |
+      Boolean is needed for authEnabled and enableEgress
+  - suppress: PostOperationIdContainsUrlVerb
+    from: SessionPools.json
+    reason: |
+      The operationId already using SessionsGeneration.
+  - suppress: PostOperationIdContainsUrlVerb
+    from: Sessions.json
+    reason: |
+      The operationId already using SessionsGeneration.
+  - suppress: MissingSegmentsInNestedResourceListOperation
```

Comment:
Not sure if these suppression are actually or not. Please re-confirm if it is not actual issue